Makefile

```diff
@@ -448,25 +448,27 @@ staging: doc/CHANGES
 	# Delete the tmp files
 	cd ${TMPDIR} && /bin/rm -rf daikon daikon.dist old_tar.txt new_tar.txt
 
-# Copy the files in the staging area to the website.  This will copy
-# all of the files in staging, but will not delete any files in the website
-# that are not in staging.
-staging-to-www: $(STAGING_DIR)
-#copy the files
+# Copy the files in the staging area to the website.
+# We do this by deleting the previous release directory and
+# renaming the staging directory to be the release directory.
+staging-to-www: $(STAGING_DIR) save-current-release
+	@echo "staging-to-www is disabled while we test new layout."
```

Comment:
I was worried about accidentally deleting to current release but I just made a backup copy instead.